src/OpenTelemetry.Collector.Azure/OpenTelemetry.Collector.Azure.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Could it be part of the existing Dependencies project? There are no extra dependencies this one brings and there is one less step for customers to do to get Azure SDK telemetry